spec/10-overview.md

```diff
@@ -1,16 +1,40 @@
 # Overview
 
-Trace context consists of a set of properties crucial for event correlation in distributed applications. Trace context positions requests in a distributed trace. If you look at a distributed trace as a graph of correlated requests - a single request can be part of multiple graphs. For example, let's say component `A` calls component `B` which in turn calls `C`. One distributed tracing system may see component `A` calling component `B`. Another may see all three components communicating. So the call from `B` to `C` will carry trace context about service `A` as well as information about its direct communication upstream to `B`.
+Trace context consists of a set of properties crucial for event
+correlation in distributed applications. Trace context positions
+requests in a <a>distributed trace</a>. If you look at a <a>distributed

```

Comment:
this is the only not a new-line modification - adding `<a></a>` twice here.

Reply:
what is the purpose of `<a></a>` without href?

Reply:
It's a way ReSpec marks mentions of definitions. https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide#definitions-and-linking. I didn't find a better way to do it that will be markdown friendly and easy to use

Reply:
This PR changes way more than just a glossary. The changes conflict with a lot of other PRs. Please make this more granular